agents/smurf_stream_simulator/smurf_stream_simulator.py

```diff
@@ -164,6 +169,7 @@ def start_background_streamer(self, session, params=None):
                 f = core.G3Frame(core.G3FrameType.Scan)
                 f['session_id'] = 0
                 f['frame_num'] = frame_num
+                f['sostream_id'] = self.stream_id
```

Comment:
How about 'stream_id' instead of 'sostream_id'? (Unless this is already established somewhere like the TOD format document?) That would keep it consistent with "stream-id", etc. elsewhere and "so" feels redundant.

Reply:
I chose 'sostream_id' to match the 'sostream_version' and 'sostream_flowcontrol' keys described here: https://github.com/simonsobs/tod2maps_docs/pull/4. I'd be fine switching but the key is already written into a lot of G3Files since it is set by the smurf-streamer... If this is really important to you we can figure out how to change it but it will be much easier to keep it as is unfortunately.

Reply:
Ah, makes sense. We should update that tod_format document with this addition.